.vs/ResxConverter/xs/UserPrefs.xml

```diff
@@ -0,0 +1,18 @@
+﻿<Properties StartupConfiguration="{6969AA12-7406-4BAB-9908-B8FDABD852F1}|Default">
```

Comment:
Should the `.vs`folder be removed and added to `.gitignore`?

Reply:
oh boy. this passed. will fix it.